### DISCUSSION TITLE
Multi model implementation

### DISCUSSION THREAD

**Discussion by erenat77** [facebookresearch/faiss#1932] [Q&A]
Can we have 2 different searches at the same time? 
For example, I want to do a keyword search at first and then want to check image similarity? 
How can we handle keyword search in FAISS (not semantic search)?

**Comment by mausch** [comment]
See the OpenSearch kNN plugin.
https://opensearch.org/docs/latest/search-plugins/knn/approximate-knn/#using-approximate-k-nn-with-filters
